common-arcana.lic

```diff
@@ -65,8 +65,8 @@ module DRCA
     true
   end
 
-  def ritual(spell)
-    DRCT.retreat
+  def ritual(spell, settings = nil)
+    DRCT.retreat(settings.ignored_npcs)
```

Comment:
Will this throw if settings is nil?

Reply:
strangely, it wont
```
>;en echo nil.something
--- Lich: exec1 active.
[exec1: ]
--- Lich: exec1 has exited.
```

Reply:
compared to the interpreter. I'm not arguing that what i have is correct or acceptable
```
2.0.0-p648 :001 > nil.something
NoMethodError: undefined method `something' for nil:NilClass
        from (irb):1
        from /home/jonas/.rvm/rubies/ruby-2.0.0-p648/bin/irb:12:in `<main>'
2.0.0-p648 :002 >
```

Reply:
Oh right, `lich.rbw` modifies the behavior of `nil`...

Reply:
Im down to refactor it to not rely on that